features/step_definitions/admin_steps.rb

```diff
@@ -108,6 +108,10 @@
   FactoryBot.create(:admin_post, title: title)
 end
 
+Given "an admin post with expired commenting" do
```

Comment:
Instead of adding this new step, you can use the existing steps for creating posts, then use [cucumber-timecop steps](https://gitlab.com/zedtux/cucumber-timecop/-/blob/master/lib/cucumber/timecop.rb) to control time, e.g. `it is currently 14 days from now`.